wiki/Wiki-Entry: Binge-watching and self control failure

```diff
@@ -0,0 +1,64 @@
+I have chosen this exact study because it focuses on binge-watching which could be described as an almost addictional use of digital media with addiction being an aspect of digital media use in the computational scoping review that immediately caught my eye. 
+
+***(R) Relationship*** between the argument in the study and the theoretical arguments put forward in the other articles 
+Prof. Dr. Reinecke is a well-known author of research that focuses on the effects of media use, thus making almost every text of his a typical text in the research area of digital media use. In “Self-regulation as a key boundary condition in the relationship between social media use and well-being”, Reinecke and his colleagues propose the idea that digital media use can have positive and negative effects on the well-being and satisfaction of intrinsic needs of an individual (Reinecke et al., 2022); very similar to the here mentioned study.
+A very interesting aspect of this paper though is, that they find that originally intrinsic motivations, such as staying connected, through extrinsic pressure can turn into feelings of being forced and thus undermining perceived autonomy (Reinecke et al., 2022, p.4). This aspect would have been an interesting perspective when looking at binge-watching behaviour.  
+As Castro and colleagues state in their paper, the introduction of Video on Demand-platforms (VoD) has transformed the ways that the audience watches TV (Castro et al., 2021). It opens new forms of self-determined digital media experiences and thus enhances “the level of control that viewers have over what they watch” (Castro et al., 2021, p. 1); completely new opportunities for autonomous digital media use.
+Interestingly, though, they find that binge-watching is, all in all, a pretty awarely used way of dealing with situations. In more than half the cases, the participants watched only for as long as they intended (Castro et al., 2021, p. 15), which plays into the idea of Granow et al. that binge-watching is also positively correlated to recovery experiences and (perceived) autonomy.
+In a wide range of research, in general, it is shown that a general feeling of lacking autonomy leads individuals to binge-watching (e.g., Masur et al., 2014) as it proposes a way of deciding for oneself how long, what, and where they wanna watch something.
+A good summary or maybe even a slight extension of the idea of Granow and her colleagues in the here presented study is that of users only being able to retrieve the positive outcomes of digital media use if they manage to actually use it in a self-controlled and moderate way so that it increases their (perceived) autonomy rather than undermining it by giving them the feeling of missing self-regulation (Hofmann et al., 2017).
```

Comment:
Can you give more examples beyond self-control that might be relevant here?

Reply:
[...] positive outcomes of digital media use if they manage to actually use it in a self-controlled and moderate way so that it increases their (perceived) autonomy rather than undermining it by giving them the feeling of missing self-regulation (Hofmann et al., 2017), while at the same time opening doors for feelings of relatedness in talking to others about watched shows, as well as enhancing compentences in handling digital aplliances.